datafusion/functions/src/string/replace.rs

```diff
@@ -96,9 +96,9 @@ fn get_replace_doc() -> &'static Documentation {
 | ABcdbaBA                                        |
 +-------------------------------------------------+
 ```"#)
-            .with_standard_argument("str", "String")
-            .with_standard_argument("substr", "Substring expression to replace in the input string. Substring expression")
-            .with_standard_argument("replacement", "Replacement substring")
+            .with_standard_argument("str", Some("String"))
+            .with_standard_argument("substr", Some("Substring expression to replace in the input string. Substring expression"))
```

Comment:
This seems odd, as it would result in "Substring expression to replace in the input string. Substring expression expression to operate on. Can be a constant, column, or function, and any combination of operators."

likely would just want that one to be .with_argument(...) with complete text.